core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -35,23 +35,24 @@ typedef float64 CellType_F64;
 #define CHECK_MEMORY_OVERFLOW(bytes)                             \
     do {                                                         \
         uint64 offset1 = (uint64)offset + (uint64)addr;          \
-        if (offset1 + bytes <= (uint64)get_linear_mem_size())    \
+        if (disable_bounds_checks                                \
```

Comment:
Will it impact the performance when `WASM_CONFIGUABLE_BOUNDS_CHECKS == 0`?

Reply:
No impact here, if WASM_CONFIGUABLE_BOUNDS_CHECKS  == 0, the disable_bounds_checks is always false, compiler is smart enough to handle it.